core/src/main/scala/org/broadinstitute/dsde/rawls/billing/BillingProjectOrchestrator.scala

```diff
@@ -113,7 +122,7 @@ class BillingProjectOrchestrator(ctx: RawlsRequestContext,
       )
       _ <- billingRepository.createBillingProject(
         RawlsBillingProject(createProjectRequest.projectName,
-                            CreationStatuses.Ready,
+                            CreationStatuses.Creating,
```

Comment:
Google impl will set this to Ready after `postCreationSteps`, Azure to CreatingLandingZone.